app/src/main/kotlin/com/music/echo/playback/MusicService.kt

```
@@ -3735,64 +3805,111 @@ class MusicService :
         secPlayer.seekTo(targetIndex, plan?.incomingStartMs ?: 0)
         if (plan != null) {
-            automixBaseParams = try { player.playbackParameters } catch (e: Exception) { PlaybackParameters.DEFAULT }
-            if (plan.tempoRatio != 1f) {
-                secPlayer.playbackParameters = PlaybackParameters(
-                    automixBaseParams.speed * plan.tempoRatio,
-                    automixBaseParams.pitch,
-                )
-            } else if (automixBaseParams != PlaybackParameters.DEFAULT) {
-                secPlayer.playbackParameters = automixBaseParams
-            }
+            val base = try { player.playbackParameters } catch (e: Exception) { PlaybackParameters.DEFAULT }
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)